src/main/java/com/target/studentinfo/exception/ErrorCode.java

```diff
@@ -2,8 +2,10 @@
 
 public class ErrorCode {
     // Error messages
-    public static final String INVALID_REQUEST_BODY = "Request body cannot be null";
-    public static final String INVALID_FIRST_NAME = "First name cannot be null";
-    public static final String INVALID_LAST_NAME = "Last name cannot be null";
-    public static final String INVALID_EMAIL_ID = "Email id cannot be null";
+    public static final String INVALID_REQUEST_BODY = "INVALID_REQUEST_BODY";
+    public static final String INVALID_FIRST_NAME = "INVALID_FIRST_NAME";
+    public static final String INVALID_LAST_NAME = "INVALID_LAST_NAME";
+    public static final String INVALID_STANDARD = "INVALID_STANDARD";
+    public static final String STUDENT_NOT_FOUND = "STUDENT_NOT_FOUND";
+
```

Comment:
Use enum instead of the string variables. For eg
```
enum ErrorCode {
    INVALID_REQUEST_BODY,
    INVALID_FIRST_NAME,
    INVALID_LAST_NAME,
    INVALID_STANDARD,
    STUDENT_NOT_FOUND
}
```
You can access them like `ErrorCode.INVALID_REQUEST_BODY`.